styx-common/src/main/java/com/spotify/styx/storage/Storage.java

```diff
@@ -116,14 +117,14 @@
    * @return true if the queried workflow is enabled
    */
   @Deprecated
-  boolean enabled(WorkflowId workflowId) throws IOException;
+  boolean isWorkflowEnabled(WorkflowId workflowId) throws IOException;
```

Comment:
These were named with the argument type in mind, like `store(Workflow)`, `delete(WorkflowId)`, `enabled(WorkflowId)`.

The names are all repetitive now with `storeWorkflow(Workflow)`.

I can see that the last one is be better as `isEnabled(WorkflowId)`, though.

If we want to rename them, then maybe the metric names in `MeteredStorage` should be aligned too.